src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) center no-repeat;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
I believe you should've copied your code from `stars` task you did before. this is not whole code. there should be modifiers as well.

Reply:
https://github.com/mate-academy/layout_catalog/pull/3562/files#r1300423105
didn't fix it